package.json

```diff
@@ -19,11 +19,6 @@
   "scripts": {
     "start": "echo \"Need to setup concurrently and start & dev scripts\" && exit 1"
   },
-  "husky": {
```

Comment:
Did we agree to remove the pre-linting?
If so, I must have missed that :)

Reply:
Nope, I removed it at one point due to errors, believing it might go away, I will add it back.

Reply:
Fixed in latest push